src/test/java/dev/codesoapbox/dummy4j/dummies/internet/UrlHostTest.java

```diff
@@ -20,13 +20,12 @@ void shouldCreateCompleteHost() {
     }
 
     @ParameterizedTest
-    @CsvSource(emptyValue = "", value = {
+    @CsvSource({
             "test, ",
```

Comment:
The space is reduntant (I think).

Reply:
removed where it didn't hurt readability